shaharyar/README.1.md

```diff
@@ -0,0 +1,4 @@
+# How to run the code
+- Go to infra directory
+- pip install -r requirements.txt
+- cdk synth && cdk deploy
```

Comment:
Read me file is good, you are using markdown style that's impressive. I would encourage you to add project title and summary of project in there too.

Reply:
Yes, I agree. I was still working on it.